plugin/tower/pkg/controller/policy/controller_test.go

```diff
@@ -439,6 +440,33 @@ var _ = Describe("PolicyController", func() {
 					)
 				})
 			})
+
+			When("create SecurityPolicy with alg protocol", func() {
+				var policy *schema.SecurityPolicy
+				var ingress, egress *schema.NetworkPolicyRule
+
+				BeforeEach(func() {
+					policy = NewSecurityPolicy(everouteCluster, false, nil, labelA, labelB)
+					ingress = NewNetworkPolicyRule("FTP", "20-80", nil, labelB, labelC)
+					egress = NewNetworkPolicyRule("TFTP", "", nil, labelA, labelC)
+					policy.Ingress = append(policy.Ingress, *ingress)
+					policy.Egress = append(policy.Egress, *egress)
+
+					By(fmt.Sprintf("create SecurityPolicy %+v", policy))
+					server.TrackerFactory().SecurityPolicy().CreateOrUpdate(policy)
+
+					By("wait for v1alpha1.SecurityPolicy created")
+					assertPoliciesNum(ctx, 1)
+				})
+				It("should generate expect policies", func() {
```

Comment:
Add extra policy inside for deleting.
We only flush CT in "conntrack" while deleting, dynamic items will remain in table "expect".